apps/aehttp/src/aehttp_helpers.erl

```diff
@@ -720,6 +728,25 @@ dry_run_txs_([#{ <<"tx">> := ETx } | Txs], Acc) ->
 dry_run_txs_([#{ <<"call_req">> := CallReq } | Txs], Acc) ->
     dry_run_txs_(Txs, [{call_req, CallReq} | Acc]).
 
+
+dry_run_tx_hash(TxHash) ->
```

Comment:
why do we need this? Can one not use the present dry_run machinery after having found the Tx via the hash (and removed signature)?

Reply:
It should be clear from the linked issue and its discussion? This is just a convenience feature - but since there is an actual use-case from SDK side, it simplifies things to provide it.

Reply:
What I meant here was... there is kind of code duplication here, can we do without that duplication?